test/callback_api.js

```diff
@@ -274,33 +274,19 @@ suite('ConfirmChannel', () => {
   });
 });
 
-suite('Error handling', () => {
-  /*
-  I don't like having to do this, but there appears to be something
-  broken about domains in Node.JS v0.8 and mocha. Apparently it has to
-  do with how mocha and domains hook into error propogation:
-  https://github.com/visionmedia/mocha/issues/513 (summary: domains in
-  Node.JS v0.8 don't prevent uncaughtException from firing, and that's
-  what mocha uses to detect .. an uncaught exception).
-
-  Using domains with amqplib *does* work in practice in Node.JS v0.8:
-  that is, it's possible to throw an exception in a callback and deal
-  with it in the active domain, and thereby avoid it crashing the
-  program.
-   */
-  if (util.versionGreaterThan(process.versions.node, '0.8')) {
-    test('Throw error in connection open callback', (done) => {
-      const dom = domain.createDomain();
-      dom.on('error', failCallback(done));
-      connect(dom.bind((_err, _conn) => {
-        throw new Error('Spurious connection open callback error');
-      }));
-    });
-  }
+describe('Error handling', () => {
+
+  it('Throw error in connection open callback', (done) => {
```

Comment:
Since amqplib no longer supports v0.8 or earlier, we no longer have to worry about ignoring the test in earlier version